docs/semgrep-secrets/rules.md

```diff
@@ -0,0 +1,202 @@
+---
+slug: rules
+append_help_link: true
+title: Rules
```

Comment:
Note that there's no need to make the title here some descriptive phrase because it's a reference doc and it's clear enough.